lib/helpers.js

```diff
@@ -101,3 +101,11 @@ export function getKeyAgreementKeyPair({verificationKeyPair}) {
   }
   return {keyAgreementKeyPair, contexts};
 }
+
+export function getMultibaseMultikeyHeader({value}) {
+  if(!value) {
+    throw new Error(
+      '"publicKeyMultibase" or "publicKeyBase58" value is required.');
+  }
+  return value?.slice(0, 4);
```

Comment:
With the check above, we don't need the check below:

```suggestion
  return value.slice(0, 4);
```

Reply:
Updated [5eac335](https://github.com/digitalbazaar/did-method-key/pull/55/commits/5eac3359fb95a394f768a1f9c62d231c96f40a5a)